config/locales/en/activerecord.yml

```diff
@@ -42,8 +42,11 @@ en:
           attributes:
             base:
               none_selected: Select if you receive any types of income
-
         legal_framework/merits_task_list:
           attributes:
             task_list:
               incomplete: You must complete every section before you can continue
+        hmrc:
+          attributes:
+            use_case:
+              blank: "Invalid use case"
```

Comment:
This branch does not appear to have the latest specs that test for these error messages (i changed in 3028 [here](https://github.com/ministryofjustice/laa-apply-for-legal-aid/pull/3707/files#diff-99733ce6351656908f1dd1f1fb9f1fc4befbf6b126ccc4bcb98644c5eb4c3f03R19) that was only merged yesterday PM).

if you rebased I am not sure they would pass as I think it needs and `inclusion:` and `blank`
```
blank: can't be blank
inclusion: Invalid use case 
```

`blank` can be left out as it is currently the default from AR anyway I think